docs/testing.rst

```diff
@@ -0,0 +1,32 @@
+.. _testing:
+
+======================
+Tests for Page Objects
+======================
+
+``web-poet`` provides :ref:`tools for testing Page Objects
+<web-poet:web-poet-testing>`. ``scrapy-poet`` projects can use a Scrapy command
+to easily generate tests::
+
+    scrapy create_po_test my_project.po.MyItemPage http://example.com/page1.html
+
+This will request the provided page, create an instance of the provided Page
+Object for this page, request its :meth:`~web_poet.pages.ItemPage.to_item`
+method and save both the Page Object dependencies and the resulting item as a
```

Comment:
Welcome to the `Page Object` naming club! 🎉 I used this naming convention as well before.😄 But @Gallaecio , @kmike , and I came to an agreement to use the lower case on this alongside `item class` in another PR (or was it from a call?).